ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.test.js

```diff
@@ -144,4 +128,13 @@ describe('AdvancedGasFeeDefaults', () => {
     fireEvent.click(checkbox);
     expect(mock).toHaveBeenCalledTimes(2);
   });
+
+  it('should not render option to set default gas options in a swaps transaction', () => {
+    render({}, { editGasMode: EDIT_GAS_MODES.SWAPS });
+    expect(
+      screen.queryByText(
+        'Always use these values and advanced setting as default.',
```

Comment:
should this be "new values" since the option wouldn't have been selected?

Reply:
yep fixed